src/components/modals/confirm-modal/README.md

```diff
@@ -0,0 +1,42 @@
+# Confirm Modal
+
+A confirmation standard modal dialog.
+
+## Usage
+
+```jsx
+import { ConfirmModal } from '@nomios/web-uikit';
+
+const handleConfirm = () => alert('Confirm');
+const handleCancel = () => alert('Cancel');
+
+<ConfirmModal
+    title="Are you sure?"
+    description="By continuing you accept our terms and conditions."
+    onConfirm={ handleConfirm }
+    onCancel={ handleCancel }
+    open />
+```
+
+You may want to use this modal with the `<ModalTrigger>` component. There are a few examples in the [Modal/Base](/?path=/story/modal-base--standalone) page.
+
+## ConfirmModal props
+
+| name | type | default | description | required |
+| ---- | ---- | ------- | ----------- | -------- |
+| title | string | | The title. Usually a question. | ✅|
```

Comment:
Can you delete the `required` column and use `*required*` on the default `column` for the sake of consistency?